tests/framework/installer/ceph_manifests_v1.2.go

```diff
@@ -1361,6 +1361,55 @@ data:
 `
 }
 
+func (m *CephManifestsV1_2) GetBlockSnapshotClassDef(snapshotClassName, namespace, systemNamespace, reclaimPolicy string) string {
+	// block snapshot was not implemented in v1.2
+	return ""
+}
+
+func (m *CephManifestsV1_2) GetFileStorageSnapshotClassDef(snapshotClassName, namespace, systemNamespace, reclaimPolicy string) string {
+	// filesystem snapshot was not implemented in v1.2
+	return ""
+}
+
+func (m *CephManifestsV1_2) GetPVCRestoreDef(claimName, snapshotName, namespace, storageClassName, accessModes, size string) string {
+	// snapshot restore to a new PVC was not implemented in v1.2
+	return ""
+}
+
+func (m *CephManifestsV1_2) GetPVCCloneDef(cloneClaimName, parentClaimName, namespace, storageClassName, accessModes, size string) string {
+	// clone pvc to a new PVC was not implemented in v1.2
+	return ""
+}
+
+func (m *CephManifestsV1_2) GetSnapshotDef(snapshotName, claimName, snapshotClassName string) string {
+	// pvc snapshot was not implemented in v1.2
+	return ""
+}
+
+func (m *CephManifestsV1_2) GetPodDef(podName, claimName, namespace, mountPath string, readOnly bool) string {
+	return `
```

Comment:
Any new methods on CephManifestsV1_2 should all return the empty string. The upgrade test won't be calling them.